packages/react/src/components/LinkWithIcon/LinkWithIcon.js

```diff
@@ -5,8 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import ArrowDown20 from '@carbon/icons-react/es/arrow--down/20';
```

Comment:
Not sure if we should be importing these icons on this level. `LinkWithIcon` should allow for positioning of the icon to be left or right, but not restricting what icons are used exactly.